### ISSUE TITLE
Query features from feature source with spatial filter may produce long request

### ISSUE THREAD

**Issue by AlenKelemen** [jumpinjackie/mapguide-rest#173]
If I use complex spatial filter (like buffer polygon with large WKT) my request may become very long, for example:
`http://localhost/mapguide/rest/library/[someSource].FeatureSource/features.geojson/schema/class?filter=Geom INSIDE GeomFromText('POLYGON(( [here comes long WKT] ))').`
Maybe we could improve this using POST instead  GET ?

**Comment by jumpinjackie** [comment]
Yes, as un-restful as that may be, it's something that should be supported.

**Comment by AlenKelemen** [comment]
I solved this with:
```
 $.ajax({
                       ```
     url: "http://" + location.hostname + "/mapguide/rest" + "/library/ppdVukovar/Data/imagisPlin.FeatureSource/features.geojson/plin/Ventil",
                            dataType: "text",
                            type: "POST",
                            data: "filter=Geom INSIDE GeomFromText('" + wktBuffer + "')",
                            headers: {
                                "X-HTTP-Method-Override": "GET"
                            },....
```
```
